appcues/src/main/java/com/appcues/debugger/DebuggerRecentEventsManager.kt

```diff
@@ -65,8 +67,12 @@ internal class DebuggerRecentEventsManager(
                 // in case something changes in the future to avoid unwanted exceptions
                 timestamp = (request.profileUpdate?.get(AutoPropertyDecorator.UPDATED_AT_PROPERTY) as Long?) ?: Date().time,
                 name = request.userId,
-                properties = request.profileUpdate?.toSortedList(),
-                identityProperties = null
+                propertySections = listOf(
+                    DebuggerEventItemPropertySection(
+                        title = contextResources.getString(R.string.appcues_debugger_event_details_properties_title),
+                        properties = request.profileUpdate?.toSortedList(),
+                    )
+                )
```

Comment:
small refactor here - where instead of having `properties` and `identityProperties` we now just have a collection of `propertySections` -- an extensible list that we can add or remove sections from as desired.  This supports adding the new section for the form inputs down on line 123.